go/test/endtoend/tabletmanager/tablet_test.go

```diff
@@ -100,3 +101,14 @@ func TestResetReplicationParameters(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, res.Rows, 0)
 }
+
+// TestGetServerStatus tests the GetServerStatus RPC
+func TestGetServerStatus(t *testing.T) {
```

Comment:
Add more tests for unknown variables, multiple variables.

Reply:
Agreed. And maybe add and test a method for getting all of them? Maybe that's what no specific variables would mean. So the rough equivalent of `show global status` vs `show global status like`.

Reply:
And while doing that, may as well add the variable for server uptime into the tests.

Reply:
Fixed